compose/cli/main.py

```diff
@@ -268,6 +268,7 @@ def pull(self, project, options):
         Usage: pull [options] [SERVICE...]
 
         Options:
+            --skip-missing          Skip unpullable images.
```

Comment:
I agree with @dnephin here around the messaging. A suggestion; how about  `Continue pulling images, irrespective of failures`

The language we use here to describe this flag is really important to be clear. We don't know that images are missing. I would prefer `--continue-on-error`